tests/unit/job/jobProcessor/JobProcessor.spec.ts

```diff
@@ -199,11 +205,16 @@ describe('JobProcessor', () => {
         .reply(200, { ...dequeuedTask })
         .persist();
 
+      // rejecting the task when it reaches max attempts
+      nock(jobManagerBaseUrl).put(updateTaskUrl).reply(200).persist();
+
+      nock(jobManagerBaseUrl).put(updateJobUrl).reply(200).persist();
+
       nock(heartbeatBaseUrl).post(`/heartbeat/${dequeuedTask.id}`).reply(200, 'ok').persist();
 
       const dequeueSpy = jest.spyOn(queueClient, 'dequeue');
 
-      const jobAndTaskType = await jobProcessor['getJobWithPhaseTask']();
+      const jobAndTaskType = await jobProcessor['getJobAndTaskResponse']();
```

Comment:
jobAndTaskType same comment -> jobAndTask

Reply:
changed to jobAndTask